plugins/system/pyparts/general.py

```diff
@@ -32,6 +36,9 @@ def get_system_info(self):
 
         if self.unattended_upgrades_enabled:
             self._output['unattended-upgrades'] = "ENABLED"
+            issue_utils.add_issue(issue_types.SystemWarning(
+                'Unattanded upgrades are enabled which can lead to ' +
+                'unexpected issues. Please disable.'))
```

Comment:
If we always recommend it be disabled, why have it at all?

[0] [1] [2] aren't that direct and appears to suggest it's good to have it enabled for security updates. Perhaps, we may need to be specific? I know we've had systems break because of unattended upgrades. Just playing devil's advocate ;-)

[0] https://canonical.my.salesforce.com/ka44K000000bwoZ
[1] https://wiki.debian.org/UnattendedUpgrades
[2] https://help.ubuntu.com/community/AutomaticSecurityUpdates

Reply:
Thanks @pponnuvel ! I have updated the message. Hopefully it's a bit more specific.